engine/crates/integration-tests/src/mocks/graphql.rs

```diff
@@ -65,159 +73,36 @@ async fn graphql_handler(headers: HeaderMap, req: GraphQLRequest) -> GraphQLResp
-                    email: "jim@example.com".into(),
-                }),
-            }),
-            PullRequestOrIssue::PullRequest(PullRequest {
-                title: "Some bot PR".into(),
-                checks: vec!["Success!".into()],
-                author: UserOrBot::Bot(Bot { id: "123".into() }),
-            }),
-            PullRequestOrIssue::Issue(Issue {
-                title: "Everythings fucked".into(),
-                author: UserOrBot::User(User {
-                    name: "The Pessimist".into(),
-                    email: "pessimist@example.com".into(),
-                }),
-            }),
-        ]
-    }
-
-    #[allow(unused_variables)]
-    async fn bot_pull_requests(&self, bots: Vec<Option<Vec<BotInput>>>) -> Vec<PullRequest> {
-        vec![
-            PullRequest {
-                title: "Creating the thing".into(),
-                checks: vec!["Success!".into()],
-                author: UserOrBot::User(User {
-                    name: "Jim".into(),
-                    email: "jim@example.com".into(),
-                }),
-            },
-            PullRequest {
-                title: "Some bot PR".into(),
-                checks: vec!["Success!".into()],
-                author: UserOrBot::Bot(Bot { id: "123".into() }),
-            },
-        ]
-    }
-
-    async fn pull_request_or_issue(&self, id: ID) -> Option<PullRequestOrIssue> {
-        if id == "1" {
-            return Some(PullRequestOrIssue::PullRequest(PullRequest {
-                title: "Creating the thing".into(),
-                checks: vec!["Success!".into()],
-                author: UserOrBot::User(User {
-                    name: "Jim".into(),
-                    email: "jim@example.com".into(),
-                }),
-            }));
-        } else if id == "2" {
-            return Some(PullRequestOrIssue::PullRequest(PullRequest {
-                title: "Some bot PR".into(),
-                checks: vec!["Success!".into()],
-                author: UserOrBot::Bot(Bot { id: "123".into() }),
-            }));
-        } else if id == "3" {
-            return Some(PullRequestOrIssue::Issue(Issue {
-                title: "Everythings fucked".into(),
-                author: UserOrBot::User(User {
-                    name: "The Pessimist".into(),
-                    email: "pessimist@example.com".into(),
-                }),
-            }));
-        }
-        None
-    }
-
-    async fn headers(&self) -> Vec<Header> {
-        self.headers
-            .clone()
-            .into_iter()
-            .map(|(name, value)| Header { name, value })
-            .collect()
-    }
-}
-
-#[derive(SimpleObject)]
-struct Header {
-    name: String,
-    value: String,
-}
-
-#[derive(SimpleObject)]
-struct PullRequest {
-    title: String,
-    checks: Vec<String>,
-    author: UserOrBot,
-}
+    // let schema = Schema::build(Query { headers }, EmptyMutation, EmptySubscription).finish();
```

Comment:
leftover?

Reply:
Yeah, spotted that in self-review, gone now